crucible-install.sh

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+# vim: autoindent tabstop=4 shiftwidth=4 expandtab softtabstop=4 filetype=bash
+# -*- mode: sh; indent-tabs-mode: nil; sh-basic-offset: 4 -*-
+
+# Installer Settings
+SYSCONFIG="/etc/sysconfig/crucible"
+DEPENDENCIES="podman git"
+INSTALL_PATH="/opt/crucible"
+GIT_REPO="https://github.com/perftool-incubator/crucible.git"
+GIT_INSTALL_LOG="/tmp/crucible-git-install.log"
+CRUCIBLE_AUTH_FILE="/root/.docker/crucible-client-server.json"
```

Comment:
I wonder if we should avoid setting a default like this.  In the proposed INSTALL.md it doesn't even mention the --auth-file option and then with this default it almost seems like it is setup to assume/hope that the user already has this setup and ready to go.  In reality, that seems highly unlikely since it is going to be specific to Crucible -- if Crucible is not installed why would they already have this?

Reply:
agreed

Reply:
agreed